### DISCUSSION TITLE
[IONOS] Don't forget to open port in their FW

### DISCUSSION THREAD

**Discussion by Aerya** [qdm12/gluetun#731] [Q&A] [answered]
Hi,

I use my own WG VPN (Angristan) and need to forward port 52920 (tcp).
I'm a bit lost on how to achieve this.
I read https://github.com/qdm12/gluetun/wiki/Firewall-options and tried it but it doesn't help despite it's reported to work.

```
today at 22:10:57|--Firewall:
today at 22:10:57   |--VPN input ports: 52920
```
I also added an IPtables (v4 only) rule on my VPS but port is still reported as closed.
`iptables -t nat -A PREROUTING -d VPS_IP -p tcp --dport 52920 -j DNAT --to-dest 10.66.66.2:52920`

Everything seems to be ok.
```
root@NUC2:/home/aerya# docker exec wireguard iptables -nvL
Chain INPUT (policy DROP 1 packets, 40 bytes)
 pkts bytes target     prot opt in     out     source               destination         
25546 3254K ACCEPT     all  --  lo     *       0.0.0.0/0            0.0.0.0/0           
55782   24M ACCEPT     all  --  *      *       0.0.0.0/0            0.0.0.0/0            ctstate RELATED,ESTABLISHED
 6340  346K ACCEPT     all  --  eth0   *       0.0.0.0/0            172.17.0.0/16       
    0     0 ACCEPT     tcp  --  wg0    *       0.0.0.0/0            0.0.0.0/0            tcp dpt:52920
    0     0 ACCEPT     udp  --  wg0    *       0.0.0.0/0            0.0.0.0/0            udp dpt:52920
```

What am I missing?

**Comment by qdm12** [comment]
Correct me if I'm wrong, but you don't want to forward the port right? You want to have this port as your Wireguard endpoint port right?

If so see https://github.com/qdm12/gluetun/wiki/Custom-provider#wireguard-only to set this port as the endpoint port.

**Comment by Aerya** [comment]
Its about rTorrent and I'm asked to be "connectable" so I need to forward this port yeah.

**Comment by qdm12** [comment]
Just use `FIREWALL_VPN_INPUT_PORTS=52920`. I didn't know Cyberghost allowed port forwarding, I'll add it to the wiki

**Comment by Aerya** [comment]
Well I don't use Cyberghost, it's a VPS with Wireguard. BTW i already tried this rule but port is still closed.

**Reply by qdm12** [reply]
You can try with FIREWALL=off to debug it.

I'm not too sure what you're trying to achieve though.

VPN server side forwarding means inbound connections are forwarded from the VPN IP address on a certain port to your VPN client (gluetun). If it still doesn't work with the firewall disabled it is a config problem outside gluetun.

**Comment by Aerya** [comment, marked as answer]
Ok. Kill me.
I totally forgot those #@! IONOS had a firewall on their VPS... "Opening the port works like a charm".

Sorry for this mess and thanks for your time (and work).

**Reply by qdm12** [reply]
No worry! I'm glad you figured it out, enjoy!
